net-im/libcommuni/libcommuni-3.5.0.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="A cross-platform IRC framework written with Qt"
+HOMEPAGE="http://communi.github.io/"
+SRC_URI="https://github.com/communi/libcommuni/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qml test +uchardet"
+
+# Compiling with USE="test" is currently broken with dev-libs/icu
+# See: https://github.com/communi/libcommuni/issues/60
+REQUIRED_USE="test? ( uchardet )"
+
+RDEPEND="dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	uchardet? ( app-i18n/uchardet )
+	!uchardet? ( dev-libs/icu:= )"
+
+DEPEND="${RDEPEND}"
+
+RESTRICT="!test? ( test )"
+
+src_prepare() {
+	default
+
+	# Don't use bundled uchardet, use instead app-i18n/uchardet
+	if use uchardet; then
```

Comment:
I think you can do this unconditionally. Conditional patching is bad since it's easy to miss mistakes when using the other configuration.

Reply:
If that is okay for you, I would like to do this. I did only conditionally, as I thought, you would find this better ;-)